microprot/snakemake/Snakefile

```diff
@@ -23,6 +23,11 @@ try:
 except KeyError:
     pass
 
+# create output directories if they don't exist
+for _out_dir_name in ['PDB', 'CM', 'AB', 'log', 'pkg']:
+    _out_dir_path = '%s/%s' % (config['MICROPROT_OUT'], _out_dir_name)
+    if not exists(_out_dir_path):
```

Comment:
you could also say `makedirs(_out_dir_path, exist_ok=True)` to always create those dirs even if they already exist